extra/utils/permissions.js

```diff
@@ -0,0 +1,26 @@
+permissions = 
+{
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    delete: [],
+    }
+}
+
+const hasPermission = (moduleName, role, permissionType) =>{
```

Comment:
need to optimise hasPermission function.
hasPermission is not working correctly.
head-trainer is included in all, so he has permission for all of the read, write and delete.